projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -259,6 +259,9 @@ public function enqueue_scripts() {
 			JETPACK__VERSION
 		);
 
+		wp_style_add_data( 'jetpack-admin-menu', 'rtl', $this->is_rtl() );
+		$this->configure_colors_for_rtl_stylesheets();
```

Comment:
Non-blocking (and personal preference): `wp_styles()->add_data( 'colors', 'rtl', $this->is_rtl() );` is simple enough, it doesn't need a separate function.

Reply:
Can we also use `wp_style_add_data` instead of `wp_styles()->add_data` so both styles (`jetpack-admin-menu` and `colors`) use the same syntax?

Reply:
>it doesn't need a separate function.

It doesn't need to be but the reason we made it a separate function is because it's responsible for fixing a core bug. We wanted to make that clear by putting it into a separate method and naming it to be more descriptive around its responsibilities.

I'll add the trac issue here also, but knowing the above are you comfortable with leaving it or would you rather us move that into the `enqueue_scripts` method?

Reply:
> We wanted to make that clear moving it and naming it to be more descriptive around its responsibilities.

That's fine 👍 We could state that in the docstring of the method to make it super clear.